Apps/W1/HybridGP/app/src/Migration/Customers/GPCustomerMigrator.codeunit.al

```diff
@@ -385,16 +390,14 @@ codeunit 4018 "GP Customer Migrator"
         HelperFunctions: Codeunit "Helper Functions";
         ClassId: Text[20];
         AccountNumber: Code[20];
-        MigrateCustomerClasses: Boolean;
     begin
-        if not GPRM00101.FindSet() then
+        if not GPCompanyAdditionalSettings.GetMigrateCustomerClasses() then
```

Comment:
I believe this was removed as a duplicate in another PR. Unsure if still needed for this PR but I am not sure which PR goes first and if the HelperFunction's have been updated for this.